telegram/ext/filters.py

```diff
@@ -116,7 +116,8 @@
 
 
 class BaseFilter:
-    """Base class for all Filters.
+    """
+    Base class for all Filters.
```

Comment:
TBH I don't really see any value in these changes - in this PR or even at all, that is.
True, some style guides suggest to start the docstring on a new line, but none of the styling tools that we currently use will report on these things.  __If__ we want to have docstrings start on a new line, I would rather like to make it a standalone issue to apply this to the whole code base.
I'm inclined to ask to revert these changes. Let's first hear what others think, though.

Reply:
I also agree to revert, they are rendered the same and [PEP 257](https://peps.python.org/pep-0257/) does not comment on this design.